people-details-sql-jdbc/src/main/java/loader/PropertiesLoader.java

```diff
@@ -0,0 +1,41 @@
+package loader;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertiesLoader {
+    private static PropertiesLoader instance;
+
+    private final Properties properties = new Properties();
+
+    private PropertiesLoader() {
+        try (InputStream inputStream = getClass().getClassLoader().getResourceAsStream("application.properties")) {
```

Comment:
application.properties seems like a magic string in the code - you could exclude this to a private static field (this applies to url and the other magic strings too)